diff_test.go

```diff
@@ -752,6 +752,51 @@ func TestSchemaDiffRoutines(t *testing.T) {
 	}
 }
 
+func TestSchemaDiffFilteredViewDiffs(t *testing.T) {
+	s1v1 := aView()
+	s1 := aSchemaWithViews("s1", &s1v1)
+
+	s2v1 := anotherView()
+	s2v2 := anAlterView()
+
+	s2 := aSchemaWithViews("s1", &s2v1, &s2v2)
+
+	assertFiltered := func(sd *SchemaDiff, expectLen int, types ...DiffType) {
+		t.Helper()
+		diffs := sd.FilteredViewDiffs(types...)
+		if len(diffs) != expectLen {
+			t.Errorf("Wrong result from FilteredViewDiffs(%v) based on count alone: expect %d, found %d", types, expectLen, len(diffs))
+		}
+		for _, diff := range diffs {
+			var ok bool
+			for _, typ := range types {
+				if diff.Type == typ {
+					ok = true
+					break
+				}
+			}
+			if !ok {
+				t.Errorf("Unexpected diff %v in result of FilteredViewDiffs(%v)", diff, types)
```

Comment:
IMO it's a little easier to parse this if you put the result at the end. Same convention used for error chaining and such.

`"FilteredViewDiffs(%v) produced unexpected result: %v"`

Reply:
Yeah, I was following the convention from the author's test in `FilteredTableDiffs` tbh.

Reply:
Fixed